tests/report/index.js

```diff
@@ -0,0 +1,21 @@
+const reporter = require('cucumber-html-reporter')
+
+const options = {
+  theme: 'bootstrap',
+  jsonFile: 'tests/report/cucumber_report.json',
+  output: 'tests/report/cucumber_report.html',
+  reportSuiteAsScenarios: true,
+  scenarioTimestamp: true,
+  launchReport: true,
+  metadata: {
+    'Ocis Version': '1.9.0',
+    'Web Version': '4.0.0',
```

Comment:
if we do it this way we always have to bump the version manually, this can be error prone.
A better way could be to read the package.json version and provision it from there.

```javascript
import {version} from 'package.json';
```